modules/how-to-plan-your-environment-according-to-application-requirements.adoc

```diff
@@ -69,3 +69,33 @@ do not. Most Java applications and applications that use huge pages are examples
 of applications that would not allow for overcommitment. That memory can not be
 used for other applications. In the example above, the environment would be
 roughly 30 percent overcommitted, a common ratio.
+
+The application pods can access a service either using environment variables which 
```

Comment:
The application Pods can access a service either by using environment variables or DNS. If using environment variables, for each active service  the variables are injected by the kubelet when a Pod is run on a node.